hermes-test-helper/src/main/java/pl/allegro/tech/hermes/test/helper/zookeeper/ZookeeperBaseTest.java

```diff
@@ -59,22 +62,31 @@ public static void tearDown() throws Exception {
         zookeeperServer.stop();
     }
 
-    public void createPath(String path) throws Exception {
-        if(zookeeperClient.checkExists().forPath(path) == null) {
+    protected void createPath(String path) throws Exception {
+        if (zookeeperClient.checkExists().forPath(path) == null) {
             zookeeperClient.create().creatingParentsIfNeeded().forPath(path);
         }
     }
 
-    public void deleteData(String path) throws Exception {
+    protected void deleteData(String path) throws Exception {
         if (zookeeperClient.checkExists().forPath(path) != null) {
             zookeeperClient.delete().deletingChildrenIfNeeded().forPath(path);
         }
     }
 
-    public void deleteAllNodes() throws Exception {
+    protected void deleteAllNodes() throws Exception {
         if (zookeeperClient.checkExists().forPath("/hermes") != null) {
             zookeeperClient.delete().guaranteed().deletingChildrenIfNeeded().forPath("/hermes");
         }
     }
 
+    protected void setupNode(String path, Object data) throws Exception {
+        createPath(path);
+        zookeeperClient.setData().forPath(path, objectMapper.writeValueAsBytes(data));
```

Comment:
You can create path and set data using `zookeeperClient.create().creatingParentsIfNeeded().forPath(path, data);`